src/cascadia/TerminalCore/TerminalDispatch.cpp

```diff
@@ -371,6 +371,19 @@ bool TerminalDispatch::EnableAlternateScroll(const bool enabled) noexcept
     return true;
 }
 
+//Routine Description:
+// Enable Bracketed Paste Mode -  this changes the behavior of pasting.
+//      See: https://www.xfree86.org/current/ctlseqs.html#Bracketed%20Paste%20Mode
+//Arguments:
+// - enabled - true to enable, false to disable.
+// Return value:
+// True if handled successfully. False otherwise.
+bool TerminalDispatch::EnableBracketedPasteMode(const bool enabled) noexcept

```

Comment:
I'd prefer this be called `EnableXtermBracketedPasteMode`, just to add more "namespacing". Thoughts?

Reply:
I mean, I'm OK with changing this. But is there any other `BracketedPasteMode` that't not Xterm-licensed?

Reply:
Honestly, I thought I had a stronger case. It looks like our other "Enable" methods are split 50/50 on whether they have the "origin" (where they came from) -- "[VT220]MouseMode", "[DEC]COLMSupport", "[Win32]InputMode". The others don't. Thanks for making the change anyway :smile: